docs/docs/cheat-sheet.md

```diff
@@ -430,7 +430,6 @@ Get the PDF: <a href="/gatsby-cheat-sheet.pdf" download>gatsby-cheat-sheet.pdf</
     Your site is now running at <code>http://localhost:8000</code>!
   </p>
   <p>
-    {/* prettier-ignore */}
     You’ll also see a second link: <code>http://localhost:8000/___graphql</code>.
     This is a tool you can use to experiment with querying your data. Learn more
     about it at <a href="https://gatsby.dev/tutorial">gatsby.dev/tutorial</a>
```

Comment:
Hello, this line is actually necessary due to the package.json scripts commands: 
"npm run prettier -- --check" or "yarn run prettier --check" 

When running that and not having {/* prettier-ignore */} causes the system to throw an error due toe the @ prettier script check. "Code style issues found in the above file(s). Forgot to run Prettier?" is the warning that occurs as well.

Thus, {/* prettier-ignore */} is necessary for this file. You can test it yourself simply by doing "npm run prettier -- --check" or "yarn run prettier --check."